fission/src/systems/scene/SceneRenderer.ts

```diff
@@ -86,11 +85,11 @@ class SceneRenderer extends WorldSystem {
         this._scene = new THREE.Scene()
 
         this._renderer = new THREE.WebGLRenderer({
-            // Following parameters are used to optimize post-processing
             powerPreference: "high-performance",
             antialias: false,
             stencil: false,
-            depth: false,
+            depth: !PreferencesSystem.getGraphicsPreferences().antiAliasing,
+            // logarithmicDepthBuffer: true,
```

Comment:
This still need to be here? Should add a comment if we want it to be a reference for later.